exchange/utils.go

```diff
@@ -64,10 +75,18 @@ func cleanOpenRTBRequests(ctx context.Context,
 		LMT:   lmtPolicy.ShouldEnforce(),
 	}
 
+	if gdpr == 1 {
+		cleanMetrics.gdprEnforced = true
+		parsedConsent, err := vendorconsent.ParseString(consent)
+		if err == nil {
+			cleanMetrics.tcfVersion = int(parsedConsent.Version())
+		}
+	}
 	// bidder level privacy policies
 	for bidder, bidReq := range requestsByBidder {
 
 		if gdpr == 1 {
+			cleanMetrics.gdprEnforced = true
```

Comment:
This doesn't seem necessary since we have it done on line 79.

Reply:
Yes, I moved this line to above, or rather copied it instead of moving. Thanks